### PR TITLE
First draft of the French version - Don't merge before review and stash.

### PR DESCRIPTION
**Pull request by natsfr**
Signed-off-by: F4IHX <f4ihx@rfporn.org>

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    "card2_amount_input": "Montant",
+    "card2_send_button": "Envoyer",
+    "send_toast_info": {
+      "success": "Paiement envoyé !",
+      "failure": "Echec de la transaction !",
+      "no_funds": "Fonds insuffisants !"
+    },
+    "scanned_balance_info": {
+      "title": "Fonds utilisables",
+      "description": "Les fonds utilisables sont disponibles immediatement par le portefeuille mobile. Après une transaction l'indication de fonds utilisables peut être temporairement desynchronisée.",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _immédiatement_ , _désynchronisée_

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  },
+  "delete_wallet_modal": {
+    "title": "Suppression",
+    "close": "Fermer",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "delete_wallet_button": "Suppression du portefeuille",
+    "delete_confirmation": {
+      "title": "Confirmation de la suppression",
+      "description": "Etes-vous sûre de vouloir supprimer ce portefeuille ?",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Etes-vous sûr_

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      "delete_button": "Supprimer",
+      "cancel_button": "Annuler"
+    },
+    "delete_outcome": {
+      "success": "Portefeuille supprimer",
+      "success_description": "Le portefeuille sélectionné a bien été supprimer !",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _supprimé_

File: src/assets/i18n/fr.json [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+      "success_button": "OK",
+      "not_found": "Portefeuille inexistant",
+      "not_found_description": "Ce portefeuille n'existe pas. Veuillez redémarrer votre application.",
+      "not_found_button": "OK",
+      "incorrect_password": "Mot de passe incorrect !",
+      "incorrect_password_description": "Saisi de mot de passe incorrect, veuillez le ressaisir.",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Saisie_

File: src/assets/i18n/fr.json [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+  },
+  "load_wallet_modal": {
+    "title": "Déchiffrement et Chargement",
+    "close": "Fermer",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "load_wallet_button": "Chargement du portefeuille",
+    "load_outcome": {
+      "incorrect_password": "Mot de passe incorrect !",
+      "incorrect_password_description": "Saisi de mot de passe incorrect, veuillez le ressaisir.",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Saisie_

File: src/assets/i18n/fr.json [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "confirm_password_input": "Confirmation du mot de passe",
+    "save_wallet_button": "Sauver"
+  },
+  "view_transaction_modal": {
+    "title": "Détail de la transaction",
+    "close": "Fermer",
+    "overview_card_title": "Aperçu",
+    "txn_id": "Indentifiant",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Identifiant_

File: src/assets/i18n/fr.json [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+  },
+  "settings_modal": {
+    "title": "Paramètres",
+    "close": "Fermer",
+    "language_input": "Langue",
+    "save_button": "Sauvegarder"
+  },
+  "shared": {
+    "warnings": {
+      "not_saved": "Votre portefeuille n'est pas sauvegarder ! Veuillez le sauvegarder maintenant !",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _sauvegardé_ (for the first one only)

File: src/assets/i18n/fr.json [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+      "not_loaded": "Portefeuille non chargé !",
+      "not_loaded_description": "Please load your wallet from the main page before continuing",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: Duplicate line here, needs to be deleted

File: src/assets/i18n/fr.json [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+      "not_found": "Portefeuille inexistant",
+      "not_found_description": "Ce portefeuille n'existe pas. Veuillez redémarrer votre application.",
+      "not_found_button": "OK",
+      "incorrect_password": "Mot de passe incorrect !",
+      "incorrect_password_description": "Saisi de mot de passe incorrect, veuillez le ressaisir.",
+      "incorrect_password_button": "OK"
+    }
+  },
+  "generate_wallet_modal": {
+    "title": "Créer et sauver",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: Check for other instances of "Sauver" and replace them with "Sauvegarder".

File: src/assets/i18n/fr.json [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+    "title": "Transactions",
+    "no_transactions": "Aucune transaction à afficher",
+    "loading_data": "Chargement des données..."
+  },
+  "delete_wallet_modal": {
+    "title": "Suppression",
+    "close": "Fermer",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "delete_wallet_button": "Suppression du portefeuille",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Supprimer le portefeuille_

File: src/assets/i18n/fr.json [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+      "button": "OK"
+    }
+  },
+  "transactions_page": {
+    "title": "Transactions",
+    "no_transactions": "Aucune transaction à afficher",
+    "loading_data": "Chargement des données..."
+  },
+  "delete_wallet_modal": {
+    "title": "Suppression",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Supprimer_

File: src/assets/i18n/fr.json [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+    "save_wallet_button": "Sauver"
+  },
+  "view_transaction_modal": {
+    "title": "Détail de la transaction",
+    "close": "Fermer",
+    "overview_card_title": "Aperçu",
+    "txn_id": "Indentifiant",
+    "hash": "Block Hash",
+    "index": "Block Index",
+    "amount": "Montant",
```
Comment (tumbly) [inline src/assets/i18n/fr.json]: _Montant de la transaction_

**Review by tumbly — CHANGES_REQUESTED** [review]
A few typos and fixes for consistency.

**Comment by tumbly** [comment]
About the english terms still left in view_transaction_modal I'm happy with them staying in english, I don't think we have a good translation for them and I find it's clearer this way.

File: src/assets/i18n/fr.json (hunk 12 above)
Comment (natsfr) [reply src/assets/i18n/fr.json]: I thought it would be a bit heavy on the list head line, any opinion ?

**Comment by natsfr** [comment]
> About the english terms still left in view_transaction_modal I'm happy with them staying in english, I don't think we have a good translation for them and I find it's clearer this way.

I agree with that, it would need half sentence to translate those correctly.

**Pull request merged**